todo-app/src/Components/Button.js

```diff
@@ -0,0 +1,35 @@
+//jshint esnext: true
+
+import React, { Component } from 'react'
+
+export default class Button extends Component {
+
+    handleClick = event => {
```

Comment:
This function isn't used. Did you mean to call `this.props.onClick` inside here?